Project.toml

```
@@ -1,6 +1,6 @@
 name = "DataStructures"
 uuid = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
-version = "0.17.6"
+version = "0.18.0"
```

```suggestion
version = "0.17.7"
```

This is not a breaking change.
Only breaking changes increment the first nonzero